<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CXProcessingParametersWindow</class>
 <widget class="QDialog" name="CXProcessingParametersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загрузить на обработку</string>
  </property>
  <layout class="QVBoxLayout" name="mCentralLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="CXTouchButton" name="mLoadBaseButton">
       <property name="text">
        <string>Взять из базы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CXTouchButton" name="mLoadButton">
       <property name="text">
        <string>Загрузить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CXTouchButton" name="mCancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CXTouchButton</class>
   <extends>QPushButton</extends>
   <header>CXTouchButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
